integration-tests/tests/workspace-dashboard.js

```diff
@@ -48,7 +110,7 @@ const setAjaxMockValues = async (testPage, namespace, name, workspaceDescription
     workspaceVersion: 'v2'
   }
   const azureWorkspacesListResult = [{
-    accessLevel: 'OWNER',
+    accessLevel: 'READER',
```

Comment:
I wanted more coverage of the menu tooltips, plus coverage of the Read-only messaging.

![image](https://user-images.githubusercontent.com/484484/167695427-dda6027a-860b-4bc3-afa8-d2aa3b550d2c.png)